spec/yaml/swiftnav/sbp/telemetry.yaml

```diff
@@ -0,0 +1,129 @@
+                  values:
+                    - 0: Pseudorange unavailable
+                    - 1: Pseudorange available
+        - pseudorange_residual:
+            type: s16
+            units: 1 dm
+            desc: Pseudorange observation residual
+        - phase_residual:
+            type: s16
+            units: 5 mm
+            desc: Carrier-phase or carrier-phase-derived observation residual
+        - outlier_flags:
+            type: u8
+            desc: Reports if observation is marked as an outlier and is excluded from the update.
+            fields:
+              - 5-7:
+                  desc: Reserved
+              - 4:
+                  desc: Measured-Doppler outlier
+                  values:
+                    - 0: Measured-Doppler accepted
+                    - 1: Measured-Doppler marked as outlier
+              - 3:
+                  desc: Computed-Doppler outlier
+                  values:
+                    - 0: Computed-Doppler accepted
+                    - 1: Computed-Doppler marked as outlier
+              - 2:
+                  desc: Carrier-phase outlier
+                  values:
+                    - 0: Carrier-phase accepted
+                    - 1: Carrier-phase marked as outlier
+              - 0-1:
+                  desc: Pseudorange outlier
+                  values:
+                    - 0: Pseudorange accepted
+                    - 1: Pseudorange marked as outlier
+                    - 2: Pseudorange marked as major outlier
+        - ephemeris_flags:
+            type: u8
+            desc: Ephemeris metadata
+            fields:
+              - 3-7:
+                  desc: Reserved
+              - 0-2:
+                  desc: Reason for ephemeris invalidity
+                  values:
+                    - 0: Valid
+                    - 1: Invalid (general status, to be extended)
+        - correction_flags:
+            type: u8
+            desc: Reserved
+        - sid:
+            type: GnssSignal
+            desc: GNSS signal identifier (16 bit)
+
+  - MSG_TEL_SV:
+      id: 0x0120
+      short_desc: Per-signal telemetry
+      desc: >
+        This message includes telemetry pertinent to satellite signals available to Starling.
+      fields:
+        - wn:
+            type: u16
+            units: weeks
+            desc: GPS week number
+        - tow:
+            type: u32
+            units: ms
+            desc: GPS Time of Week
+        - n_obs:
+            type: u8
+            desc: >
+              Total number of observations. First nibble is the size
+              of the sequence (n), second nibble is the zero-indexed
+              counter (ith packet of n)
+        - origin_flags:
+            type: u8
+            desc: Flags to identify Starling component the telemetry is reported from.
+            fields:
+              - 0-7:
+                  desc: Starling component
+                  values:
```

Comment:
My thinking for using `standalone` and `differential` was to keep the names generic and not to include Starling-specific filter names (`SPP`, `RTK` and other) since the names are customer-facing.

No strong opinion here though, would you prefer `SPP` and `RTK` better @richarddeurloo ?

Reply:
My comment was not about the values, but about the descriptions. I would not include the word `Starling` in there. My suggestions would be: 
```suggestion
            desc: Flags to identify the filter type from which the telemetry is reported.
            fields:
              - 0-7:
                  desc: Filter type:
                  values:
```

Reply:
makes total sense, thank you for clarification!